src/styleguide/routes/r-tooltips.vue

```diff
@@ -6,9 +6,11 @@
         Disable/Enable Tooltip
       </h2>
 
-      <div :class="b('link')">
-        <span v-tooltip:[position].mouseover="'Content for the tooltip'">Hover me to see the tooltip.</span>
-      </div>
+      <ul>
+        <li :class="b('link')">
```

Comment:
suggestion: I feel like `link` might not be the best name for the `<li>` element showcasing the tooltip. How about `item` or `element` for better clarity in this context?